src/frontend/app/shared/components/list/list-types/cf-spaces/cf-space-card/cf-space-card.component.spec.ts

```diff
@@ -0,0 +1,66 @@
+// import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
That's one way to get the test to pass! 👍  Lets uncommented and see if it works.